Gemfile

```diff
@@ -164,6 +164,7 @@ group :test do
   gem "factory_bot_rails"
   gem "rspec-sidekiq"
   gem "simplecov", require: false
+  gem "timecop"
```

Comment:
What are you adding timecop for? I'd prefer to avoid timecop, if at all possible, either by using `be_within(5.seconds).of xxxx` or [using the built in rails helpers](https://andycroll.com/ruby/replace-timecop-with-rails-time-helpers-in-rspec/) (full disclosure, I haven't tried the helpers).

That said, maybe there is a need for it, in which case, go for it.

Reply:
This is the test case that prompted it:

```rb
# spec/models/theft_alert_spec.rb L19-32 (3989a5d4)

it "sets begin and end times, flips the status" do
  theft_alert = FactoryBot.create(:theft_alert_paid)
  duration = theft_alert.theft_alert_plan.duration_days
  now = nil

  Timecop.freeze do
    now = Time.current
    theft_alert.begin!(facebook_post_url: "https://facebook.com")
  end

  expect(theft_alert.begin_at).to eq(now.beginning_of_day)
  expect(theft_alert.end_at).to eq(now.end_of_day + duration.days)
  expect(theft_alert.status).to eq("active")
end
```

Reply:
I don't understand why you need to freeze time for that?

Separately - I don't understand why you would do a start time of beginning of day instead of the  current time.

Reply:
Vestigial -- removed in https://github.com/bikeindex/bike_index/pull/918/commits/6ede7e8528bd971c7f3612896b9dd35dd6539c58

> Separately - I don't understand why you would do a start time of beginning of day instead of the current time.

If the time meaningful, let's not truncate it